docs/src/style.md

```diff
@@ -480,29 +480,29 @@ the call chain. Avoid this situation or handle it with a helpful error message.
 
 Bad:
 ```julia
-internal_function(x::Integer) = x + 1
-# The user sees a MethodError for internal_function when calling
+_internal_function(x::Integer) = x + 1
```

Comment:
This change is not a typo fix and needs more discussion. There's currently no section in the style guide on how to name internal functions. I would separate this out from the other changes you're proposing.

Reply:
I `internal_function` shouldn't be [exportet](http://www.juliaopt.org/JuMP.jl/v0.21.1/style/#Exported-and-non-exported-names-1), hence I updated its name.

Reply:
I retract my comment, good use of citing the style guide to support your argument :)